bundles/org.openhab.binding.warmup/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="warmup"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="my-warmup">
+		<label>My Warmup Account</label>
+		<description>Connection to the https://my.warmup.com site</description>
+		<category>WebService</category>
+		<config-description>
+			<parameter name="username" type="text" required="true">
+				<label>Username</label>
+				<description>Username for my.warmup.com</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for my.warmup.com</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" required="true">
+				<label>Refresh Interval</label>
+				<description>Interval in seconds between automatic refreshes</description>
+				<default>300</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="room">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="my-warmup"/>
+		</supported-bridge-type-refs>
+
+		<label>Room</label>
+		<description>Warmup 4iE Device controlling a room</description>
+		<category>RadiatorControl</category>
+
+		<channels>
+			<channel id="currentTemperature" typeId="currentTemperature"/>
+			<channel id="targetTemperature" typeId="targetTemperature"/>
+			<channel id="overrideRemaining" typeId="overrideRemaining"/>
+			<channel id="runMode" typeId="runMode"/>
+		</channels>
+
+		<representation-property>serialNumber</representation-property>
```

Comment:
You used the Id in the discovery service. The serial number might be more reasonable.

Reply:
Thanks - I think this is all aligned now